prime-router/src/main/resources/metadata/hl7_mapping/receivers/STLTs/PA/PA-receiver-transform.yml

```diff
@@ -0,0 +1,52 @@
+hl7Class: ca.uhn.hl7v2.model.v251.message.ORU_R01
+
+extends: classpath:/metadata/hl7_mapping/ORU_R01/ORU_R01-base.yml
+
+constants:
+  MSH: '/MSH'
+  PID: '/PATIENT_RESULT/PATIENT/PID'
+  ORC: '/PATIENT_RESULT/ORDER_OBSERVATION/ORC'
+  OBR: '/PATIENT_RESULT/ORDER_OBSERVATION/OBR'
+
+elements:
+  - name: pa-patient-race-coding-system
+    resource: 'Bundle.entry.resource.ofType(Patient).extension("http://ibm.com/fhir/cdm/StructureDefinition/local-race-cd").value.coding'
+    condition: '%resource.code.exists()'
+    value: [ '%resource.system.getCodingSystemMapping()' ]
+    hl7Spec: [ '/PATIENT_RESULT/PATIENT/PID-10-3' ]
+
+  - name: pa-sending-application-namespace-id
+    value: [ '"CDC PRIME - Atlanta, Georgia (Dekalb)"' ]
+    hl7Spec: [ '%{MSH}-3-1']
+
+  - name: pa-sending-application-universal-id
+    value: [ '"2.16.840.1.114222.4.1.237821"' ]
+    hl7Spec: [ '%{MSH}-3-2' ]
+
+  - name: pa-sending-application-universal-id-type
+    value: [ '"ISO"' ]
+    hl7Spec: [ '%{MSH}-3-3' ]
+
+  - name: pa-receiving-application-namespace-id
+    value: [ '"PA-ELR"' ]
+    hl7Spec: [ '%{MSH}-5-1' ]
+
+  - name: pa-receiving-facility-namespace-id
+    value: [ '"PADOH"' ]
+    hl7Spec: [ '%{MSH}-6-1' ]
+
+  - name: pa-sending-facility-namespace-id
```

Comment:
Where do you see these values to be filled?  Is it in the original message?

Reply:
Yes